snippet-extractor-output/snippets.yaml

```diff
@@ -1,7 +1,7 @@
 'Excel.Application#suspendScreenUpdatingUntilNextSync:member(1)':
   - >-
     // Link to full sample:
-    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/master/samples/excel/90-scenarios/performance-optimization.yaml
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/master/samples/office-js-snippets/samples/excel/90-scenarios/performance-optimization.yaml
```

Comment:
Why did all the URLs in this file change? We should investigate that, because that doesn't seem right.

Reply:
I have no idea but I can just undo this and keep an eye out for it in the future.